### DISCUSSION TITLE
Lesson 14: Adding more coverage to the tests

### DISCUSSION THREAD

**Discussion by Robschuh** [Cyfrin/foundry-full-course-cu#1862] [General]
Hi there, I am trying to figure out why there are big differences in the coverage results if we are using the same code and the same tests.

**Video coverage** 
![coverage-video](https://github.com/Cyfrin/foundry-full-course-f23/assets/142880927/4054e631-52a2-4f25-a77c-d884b32ba338)

**`My coverage**
![me-coverage](https://github.com/Cyfrin/foundry-full-course-f23/assets/142880927/923818c8-b69a-4775-8f49-b99d1649d3da)

**Comment by urosognjenovic** [comment]
I don't think you have the same code.

The video coverage shows that the contract ```FundMe.sol``` has 7 functions and yours has 10. In the video there're 16 lines of code and in your contract there're 24 lines of code. There are too many differences to expect the same coverage.

**Reply by Robschuh** [reply]
Yeah, you are 100% right. Maybe I need to follow Patrick's advice, take a break, and go for a coffee or an ice cream :)
THANKS

**Comment by EngrPips** [comment]
Those breaks are always important!
